en/components/hyperlink.md

```diff
@@ -20,9 +20,9 @@ The Hyperlink comes in two different sizes matching the available sizes for para
 
```

Comment:
```suggestion

## Url

Specify the link you want the hyperlink to open when clicked via the Url override. This will have no visibility in your design but will be taken into account by the code generation services.

```